content/docs/start/model-registry/model-cicd.md

```diff
@@ -8,7 +8,7 @@ description:
 
 # Get Started: Using and Deploying Models
 
-In the [model registry chapter](/doc/start/model-management/model-registry) we
+In the [model registry chapter](/doc/start/model-registry/model-registry) we
```

Comment:
This should not be called the "model registry chapter" any more. What is described here is now in the two preceding chapters, so I would just say that

```suggestion
In the previous two chapters we
```